projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -386,6 +386,9 @@ function wpcom_launchpad_get_task_definitions() {
 				// that are not in the Customer Home page. We should find a better way to handle this.
 				return '/domains/add/' . $data['site_slug_encoded'] . '?from=my-home';
 			},
+			'body_context'         => array( 'launchpad-navigator' ),
+			'actions_callback'     => 'wpcom_launchpad_domain_customize_actions',
+			'content_callback'     => 'wpcom_launchpad_domain_customize_content',
```

Comment:
It feels like it may be worth centralising this logic a bit so there is a single callback function to build the content, actions, and relevant contexts, along the lines of the long-term question I had above. Instead of having three separate pieces of logic, might it be simpler to have a single `extended_content_callback` that's responsible for returning the contexts, actions, and content?